decentraland/unity-renderer
# refactor: move things out of shared/ root

## What does this PR change?
Drops unused code, favors `lib/`, removes code from `packages/shared` in the browser-interface.

## How to test the changes?
1. Launch the explorer
2. Ensure all functionality is still there
